android_test/app/src/main/java/org/stellar/javastellarsdkdemoapp/AndroidSdkProvider.kt

```diff
@@ -0,0 +1,24 @@
+package org.stellar.javastellarsdkdemoapp
```

Comment:
once https://github.com/overcat/java-stellar-sdk-android-spi is published to jitpack, this can go away correct, and change this example to have dep on that instead?